go/client/cmd_simplefs_list.go

```diff
@@ -43,18 +43,18 @@ func NewCmdSimpleFSList(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 
 // HandleTopLevelKeybaseList - See if this is either /keybase/public or /keybase/private,
 // and request favorites accordingly.
-func (c *CmdSimpleFSList) HandleTopLevelKeybaseList() (bool, error) {
+func (c *CmdSimpleFSList) HandleTopLevelKeybaseList(path keybase1.Path) (bool, error) {
```

Comment:
Isn't the whole HandleTopLevelKeybaseList superfluous as the backend can do this? (or does it have issues?)